python/lsst/meas/extensions/piff/piffPsf.py

```diff
@@ -81,6 +81,13 @@ def _doComputeKernelImage(self, position, color):
     def _doComputeBBox(self, position, color):
         return self._doBBox(Point2I(0, 0), center=True)
 
+    def getAveragePosition(self):
+        if not hasattr(self, '_averagePosition'):
```

Comment:
In these cases I usually set `self._averagePosition = None` in `__init__`, and then check if it is `None`.  This is ever so marginally faster.